src/Common/src/Interop/Windows/mincore/Interop.GetNamedPipeInfo.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        internal static extern bool GetNamedPipeInfo(
```

Comment:
These four signatures are identical, except that for each one which parameter is passed as "out int" instead of IntPtr changes.  Could we just collapse these all to a single definition where "out int" is used for all of the parameters?  I realize we'd need to then change the call sites, but would that be terrible?